working/1426-extension-types/feature-specification-views.md

```diff
@@ -746,12 +774,12 @@ A compile-time error occurs if a view _DV_ has two superviews `V1` and
 declarations, and _DV_ does not declare a member named _m_.
 
 *In other words, if two different declarations of _m_ is inherited
-from two superviews then the subview must override _m_, either
-explicitly or via an export declaration. The so-called diamond
-inheritance pattern can create the case where two superviews have an
-_m_, but they are both declared by the same declaration (so `V` is a
-subview of `V1` and `V2`, and both `V1` and `V2` are subviews of `V3`,
-and `V3` declares _m_, in which case there is no conflict in `V`).*
+from two superviews then the subview must resolve the conflict. The
+so-called diamond inheritance pattern can create the case where two
+superviews have an _m_, but they are both declared by the same
+declaration (so `V` is a subview of `V1` and `V2`, and both `V1` and
+`V2` are subviews of `V3`, and `V3` declares _m_, in which case there
+is no conflict in `V`).*
```

Comment:
Hmm, in order for this to be sound, I think we also need to require that each transitive superview `V1` of _DV_ must have the same type arguments.  E.g. this is not allowed:

```dart
view class V1<T> {
  void m(T t) {}
}
view class V2 implements V1<int> {}
view class V3 implements V1<String> {}
view class DV implements V2, V3 {}
```

Reply:
Certainly! Done.